### DISCUSSION TITLE
Game save data locations database from PC Gaming Wiki

### DISCUSSION THREAD

**Discussion by Danik1601** [DavidDeSimone/OpenCloudSaves#106] [Ideas]
There is a great tool [Ludusavi](https://github.com/mtkennerly/ludusavi) that uses [Ludusavi manifest](https://github.com/mtkennerly/ludusavi-manifest) to determine different games' saves locations which is compiled from data on PCGamingWiki, along with accessing the Steam API for game installation directory names. Is it possible to do something similar in your project instead of contributing each game manually?

**Comment by DavidDeSimone** [comment]
I think this is a good idea, and is something I will explore.
